torch/jit/_recursive.py

```diff
@@ -407,14 +419,20 @@ def init_fn(script_module):
         # Wrap the original to propagate docstrings and such.
         # TODO: we don't currently do this functions that are recursively
         # compiled, we should.
-        script_method = functools.wraps(stub.original_method)(script_method)
+        script_method = functools.wraps(method_stub.original_method)(script_method)
 
         # Add the methods to the script_module directly. This ensures they will
         # be found first when `name` is looked up (as opposed to the stubs or
         # nn.Module.forward)
         script_module.__dict__[name] = script_method
 
 
+    for property_stub in property_stubs:
+        property_name = property_stub.def_.name().name
+        fget = cpp_module._get_method(property_stub.def_.getter_name().name)
+        fset = cpp_module._get_method(property_stub.def_.setter_name().name)
```

Comment:
don't we need to handle a setter not begin set ?

Reply:
Good catch, fixed.